src/editor/components/DraggableModuleWithMenu.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import {
+  Platform,
+  ActionSheetIOS,
+  UIManager,
+  findNodeHandle,
+  View,
+  TouchableOpacity,
+  StyleSheet,
+  Text,
+  TextInput,
+} from 'react-native';
+import { ModuleType } from '../../services/constants';
+import { TextStyle } from '../../Styles.config';
+
+// Modified from https://github.com/izzisolomon/react-native-options-menu to handle onLongPress and to suit our needs
```

Comment:
If you wanna see what the native menu is supposed to look like, see [here](https://github.com/izzisolomon/react-native-options-menu). I copied and modified this very smol package so that the on press menu + drag + textinput could work together. 🗿